packages/components/Shimmer/src/Shimmer.tsx

```diff
@@ -0,0 +1,98 @@
+/** @jsx withSlots */
+import { compose, UseSlots, buildProps, mergeProps, withSlots } from '@fluentui-react-native/framework';
+import { ViewProps } from 'react-native';
+import { IViewProps } from '@fluentui-react-native/adapters';
+import * as React from 'react';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+
+const shimmerName = 'Shimmer';
+
+const NativeShimmerView = ensureNativeComponent('MSFShimmerView');
+
+/**
+ * ViewProps props, based off of the standard react-native ViewProps with some new extensions
+ */
+export type NativeShimmerViewProps<TBase = IViewProps> = TBase & {
+  /**
+   * The alpha value of the center of the gradient in the animation
+   */
+  shimmerAlpha?: number;
+
+  /**
+   * The width of the gradient in the animation
+   */
+  shimmerWidth?: number;
+
+  /**
+   * Angle of the direction of the gradient, in radian. 0 means horizontal, Pi/2 means vertical.
+   */
+  shimmerAngle?: number;
+
+  /**
+   * Speed of the animation, in point/seconds.
+   */
+  shimmerSpeed?: number;
+
+  /**
+   * Delay between the end of a shimmering animation and the beginning of the next one, in seconds
+   */
+  shimmerDelay?: number;
+
+  /**
+   * Color to tint the shimmer boxes. Defaults to the Fabric default color.
+   */
+  viewTintColor?: string;
+
+  /**
+   * Corner radius on each view.
+   */
+  cornerRadius?: number;
+};
+
+/**
+ * Shimmer Token is a flat list of all the properties we can set on the native view
+ */
+export type ShimmerToken = NativeShimmerViewProps;
```

Comment:
I would reverse these and declare ShimmerTokens above, then have the props include the base reference. The way you have it set up here the tokens include all ViewProps as well.